### ISSUE TITLE
Add default image of data umbrella logo for events that don't have a url

### ISSUE THREAD

**Issue by dashcraft** [data-umbrella/event-board-web#180]

**Comment by jackwkinsey** [comment]
Is the ask on this issue to swap out the default thumbnail image?

Essentially from
<img src="https://user-images.githubusercontent.com/5323147/190276077-0443cc70-e6b6-4417-b672-ead6a795fcd8.jpg" width="300px" />

to
<img src="https://user-images.githubusercontent.com/5323147/190276123-1254e1c7-1e8b-4390-b68d-6ad199f87350.jpg" width="300px"/>

I did notice that some events have broken image links:
![Image](https://user-images.githubusercontent.com/5323147/190276581-a485db3d-4506-4fdb-89ac-86617f1ddca4.jpg)

But in the few cases I've found of those, the problem isn't a missing image URL, it's bad data. I.e. the `imageUrl` property is set to a link that isn't an image, e.g. `imageUrl: "https://2022.djangocon.us/"`. Which causes the broken image logo to be shown. That's a bit trickier to detect.
